lib/taskana-core/src/main/java/pro/taskana/spi/taskpreprocessing/api/CreateTaskPreprocessor.java

```diff
@@ -0,0 +1,13 @@
+package pro.taskana.spi.taskpreprocessing.api;
```

Comment:
I would prefer the more simple package pro.taskana.spi.task.
And do we really need ".api"? Isn't SPI already enough?

Reply:
changed the package to pro.taskana.spi.task. api/internal packages to be consistent with history and task trouting spis. Do you want all of them restructured to without api/internal packages?